iOS Example Frame SPM/iOS Example Frame SPM.xcodeproj/project.pbxproj

```diff
@@ -1235,8 +1235,8 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/checkout/frames-ios";
 			requirement = {
-				branch = "update/security-code-component-updates";
-				kind = branch;
+				kind = exactVersion;
```

Comment:
What is the branch here ?

Reply:
normally I was using the branch name for the tests to pass before merging them into develop. Now, we can take it back otherwise `develop` to `main` branch won't pass. Then while making the release, we'll update the version number